docs/magento2/search-engine.mdx

```diff
@@ -282,6 +282,16 @@ not hesitate to ask your Attraqt CSM.
 When you get the ✔️ as a response from the script, you are ready to use Attraqt
 with Front-Commerce.
 
+### Facets
+
+Due to facet IDs not being editable in Attraqt, facets created in your Attraqt
+console must be based on your Magento attributes exact names.
```

Comment:
**question:** wouldn't it become a platform-agnostic feature?

**suggestion:** if it's the case I suggest to keep platform specific terms to a bare mimimum
```suggestion
console must be based on your eCommerce  platform attributes exact names.
```

Reply:
Ok, not blocking … it can be merged!